README.md

```diff
@@ -1,40 +1,86 @@
 # ChatKNML
 
+[![KNML](https://i.imgur.com/GLpXodx.png)](https://knml.edu.pl/)
+
+[![Python](https://img.shields.io/badge/Python-3.11%2B-blue)](https://www.python.org/)
+[![Docker](https://img.shields.io/badge/Docker-24.0%2B-blue)](https://www.docker.com/)
+[![Poetry](https://img.shields.io/badge/Poetry-1.6%2B-blue)](https://python-poetry.org/)
+[![Django](https://img.shields.io/badge/Django-3.2%2B-green)](https://www.djangoproject.com/)
+[![Discord.py](https://img.shields.io/badge/Discord.py-2.3.2%2B-blue)](https://discordpy.readthedocs.io/en/stable/)
+[![PostgreSQL](https://img.shields.io/badge/PostgreSQL-16%2B-blue)](https://www.postgresql.org/)
+[![Git](https://img.shields.io/badge/Git-2.40%2B-red)](https://git-scm.com/)
+[![MIT License](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
 ## Getting started
 
+### First setup
+
+1. **Clone the repository**:
+
+    ```sh
+    git clone git@github.com:knmlprz/ChatKNML.git
+    ```
+
+2. **Navigate to the project directory**:
+
+    ```sh
+    cd ChatKNML
+    ```
+
+3. **Create a new branch for your development work**:
+
+    ```sh
+    git checkout -b {your_branch_name}
+    ```
+
+4. **Make sure you are working on the correct branch**:
+
+   ```sh
+    git status
+    ```
```

Comment:
**nitpick:** I'd drop this bold, if everything's bolded - nothing is

Reply:
+1\